### PR TITLE
BaseSell prices and model IDs update.

### PR DESCRIPTION
**Pull request by ghost**
**_I affirm:_**
- [x] that I agree to Project Topaz's [Limited Contributor License Agreement](http://project-topaz.com/blob/release/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I've _tested my code_ since the last commit in the PR, and will test after any later commits

Hello.

Updated and added some BaseSell prices as well as a lot of model IDs based on Lelia's work (thanks to him). Wasn't able to update things from his latest archive (from today) because he added item mods and such. I don't want to mess up things I'm not familiar with.

Thanks!

### CODE REVIEW

File: sql/item_equipment.sql [hunk 1]
```diff
@@ -14159,17 +14159,17 @@ INSERT INTO `item_equipment` VALUES (28644,'ogapepo_cape',99,0,1753628,0,0,0,327
...
+INSERT INTO `item_equipment` VALUES (28651,'metal_slime_shield',1,0,4194303,650,1,0,2,0);
 INSERT INTO `item_equipment` VALUES (28652,'hatchling_shield',1,0,4194303,651,1,0,2,0);
 INSERT INTO `item_equipment` VALUES (28653,'mundus_shield',1,0,4194303,644,1,0,2,0);
 INSERT INTO `item_equipment` VALUES (28654,'beatific_shield_+1',99,119,2385,646,1,0,2,0);
-INSERT INTO `item_equipment` VALUES (28655,'slime_shield',1,0,4194303,0,1,0,2,0);
+INSERT INTO `item_equipment` VALUES (28655,'slime_shield',1,0,4194303,648,1,0,2,0);
 INSERT INTO `item_equipment` VALUES (28656,'eminent_shield',99,117,193,25,3,0,2,0);
 INSERT INTO `item_equipment` VALUES (28657,'kaidate',99,109,2529,32,4,0,2,0);
-INSERT INTO `item_equipment` VALUES (28658,'sors_shield',99,109,3719388,0,2,0,2,0);
+INSERT INTO `item_equipment` VALUES (28658,'sors_shield',99,109,3719388,36,2,0,2,0);
```
Comment (59blargedy) [inline sql/item_equipment.sql:14172]: just spotchecked this out - it's showing as a standard round shield vs the blue sors shield. I still think having a model id is better than not but this one is not fully correct.

**Review by 59blargedy — APPROVED** [review]
spot checked a few, overall reviewed for missing commas, etc. looks good. (no idea why this wouldnt build)

**Comment by ibm2431** [comment]
Sometimes Travis has issues setting up and authenticating the packages it needs to build, and needs to be kicked. Can usually click the build status and read the output.

**Review by ibm2431 — APPROVED** [review]
レリアありがとう！

Thanks Nyu!

**Pull request merged**